### PR TITLE
Translate some missing messages (Options > General) to Czech language

### PR DESCRIPTION
**Pull request by michaeldojcar**

### CODE REVIEW

**Comment by michaeldojcar** [comment]
I have corrected also few typos.

File: cs.json [hunk 1]
```diff
@@ -267,23 +267,23 @@
...
 			"button-remove-commercial-license": "Odebrat",
 			"option-catalyst": "Catalyst license",
-			"option-catalyst-desc": "You currently have the {{tier}} Catalyst license. Thanks for your support!",
-			"option-catalyst-desc-no-license": "You don't have a Catalyst license right now. Catalyst license is a one-time donation to support Obsidian development.",
-			"label-app": "App",
-			"label-add-own-language": "See how to add your language to Obsidian."
+			"option-catalyst-desc": "Nyní máte aktivní Catalyst licenci {{tier}}. Děkujeme za podporu!",
+			"option-catalyst-desc-no-license": "Nyní nemáte aktivní Catalyst licenci. Koupí Catalyst licence jednorázově podpoříte vývoj aplikace Obsidian.",
+			"label-app": "Aplikace",
+			"label-add-own-language": "Podívejte se jak přidat váš jazyk do Obsidianu."
```
Comment (joethei) [inline cs.json:273]: Please don't translate/alter the word "Obsidian"

**Review by joethei — CHANGES_REQUESTED** [review]

**Comment by michaeldojcar** [comment]
Is it better now? I fix also some older Obsidian mentions, where the word Obsidian was inflected in other cases.

File: cs.json (hunk 1 above)
Comment (michaeldojcar) [reply cs.json]: Is it better now? I fixed also some older Obsidian mentions, where the word Obsidian was inflected in other cases.

**Pull request merged**